specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```diff
@@ -0,0 +1,903 @@
+        },
+        "objectFailedCount": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Total number of objects where task operation failed when was attempted. Filter options such as objectFailedCount eq 0 and other comparison operators can be used as described for Numerical properties in https://learn.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#supported-comparison-operators"
+        },
+        "objectsSucceededCount": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Total number of objects where task operation succeeded when was attempted.Filter options such as objectsSucceededCount gt 150 and other comparison operators can be used as described for Numerical properties in https://learn.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#supported-comparison-operators"
+        },
+        "runStatusError": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Well known Azure Storage error code that represents the error encountered during execution of the run instance."
+        },
+        "runStatusEnum": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "InProgress",
+            "Finished"
+          ],
+          "x-ms-enum": {
+            "name": "RunStatusEnum",
+            "modelAsString": true
+          },
+          "description": "Represents the status of the execution."
+        },
+        "summaryReportPath": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Full path to the verbose report stored in the reporting container as specified in the assignment execution context for the storage account. "
+        },
+        "taskId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Storage Task Arm Id."
+        },
+        "taskVersion": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Storage Task Version"
+        },
+        "runResult": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "RunResult",
+            "modelAsString": true
+          },
+          "description": "Represents the overall result of the execution for the run instance"
+        }
+      },
+      "description": "Storage task execution report for a run instance."
+    }
+  },
+  "parameters": {
+    "ResourceGroupName": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the resource group within the user's subscription. The name is case insensitive.",
+      "pattern": "^[-\\w\\._\\(\\)]+$",
+      "minLength": 1,
+      "maxLength": 90,
+      "x-ms-parameter-location": "method"
+    },
```

Comment:
Use this from v5 common types.

Reply:
@visingla-ms kindly take a look, addressed all comments